### ISSUE TITLE
Can't connect to localhost 3000

### ISSUE THREAD

**Issue by norahnerd** [ctubio/Krypto-trading-bot#146]
so I have K.js up and running, I can see it in the activity monitor, I'm not getting any errors when I run `make start` ... but I cannot connect to it at all. I've tried connecting to https://localhost:3000 on Firefox and Chromium and each cannot connect.. what do I need to do to get this working?

**Comment by ctubio** [comment]
maybe read the documentation? it says:

To report errors open a new issue only after collecting all the relevant log messages (run `nodejs K.js` to see the output).

**Comment by ctubio** [comment]
btw it also says that issues are only for bugs or new featues, consider the support or the irc channel instead
